docs/AWSAdapter.org

```diff
@@ -13,37 +13,96 @@ The =AWSAdapter= is an adapter for the AWS platform.
 
 * Requirements
   1. =AWSAdapter= will create a VM on AWS based on the specification
-     given in the lab spec for a particular lab.
+     given in the lab spec for a particular lab. 
      
-  2. =AWSAdapter= will copy the =VMManager= component and start the
-     =VMManager= service on port 9089 on the newly created VM.
+  2. =AWSAdapter= will copy the =VMManager= component to the newly
+     created VM and start the =VMManager= service on port 9089.
 
 * Design
 
-** General Adapter Design 
+** General Adapter Design (REPHRASE) 
 According to ADS, every adapter must have certain interfaces. These
```

Comment:
In the current design of ADS, the interfaces of an adapter is defined and the adapter is enforced to implement these interfaces.